runtimes/eoapi/stac/eoapi/stac/config.py

```diff
@@ -44,6 +44,7 @@ class Settings(config.Settings):
     pgstac_secret_arn: Optional[str] = None
 
     titiler_endpoint: Optional[str] = None
+    enable_transaction: bool = False
```

Comment:
should we add some kind of defense mechanism in the cdk app to make sure no one turns this on? :laughing:

Reply:
I don't want to over engineer this but feel free to make it :-)